### PR TITLE
docs(v2): 2020 recap blog post

### PR DESCRIPTION
**Pull request by slorber**
## Motivation

https://deploy-preview-4020--docusaurus-2.netlify.app/classic/blog/2020/01/12/docusaurus-2020-recap/

### CODE REVIEW

File: website-1.x/blog/2020-01-12-docusaurus-2020-recap.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+We even saw the **[first right-to-left](https://datagit.ir/)** Docusaurus 2 site published:
+
+![Datagit screenshot](/img/blog/2020-recap/datagit-rtl-screenshot.png)
+
+## GitHub Activity
+
+- **Stars**: 14632 -> 20946 (+43.2% y/y)
+- **Total Contributors**: 303 -> 512 (+68.9% y/y). Most of which are non-Facebook contributors
+- **Daily npm Downloads**: ??? TODO
```
Comment (yangshun) [inline website-1.x/blog/2020-01-12-docusaurus-2020-recap.md:70]: - **Weekly npm Downloads**: 2356 -> 25592 (+TODO y/y)

**Review by yangshun — COMMENTED** [review]
I feel we should give MLH special mention as its own category.

File: website-1.x/blog/2020-01-12-docusaurus-2020-recap.md (hunk 1 above)
Comment (slorber) [reply website-1.x/blog/2020-01-12-docusaurus-2020-recap.md:70]: thanks.
Wonder what tools you use to get these data?

Comment (yangshun) [reply website-1.x/blog/2020-01-12-docusaurus-2020-recap.md:70]: Just the npm package page and the repo insights tab, nothing special. I wanted to get more data via our internal OSS repo dashboards but they're down at the moment. I guess these numbers are sufficient anyway :D

**Review by yangshun — APPROVED** [review]

**Review by lex111 — APPROVED** [review]

**Comment by JoelMarcey** [comment]
> I feel we should give MLH special mention as its own category.

Did we remove the MLH call out in the post? I only see one mention of it in a bullet under Media. Or am I missing something?

**Comment by JoelMarcey** [comment]
Nevermind my previous comment. I just totally missed it.

File: website-1.x/blog/2020-01-12-docusaurus-2020-recap.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+## GitHub Activity
+
+- **Stars**: 14632 -> 20946 (+43.2% y/y)
+- **Total Contributors**: 303 -> 512 (+68.9% y/y). Most of which are non-Facebook contributors
+- **Weekly npm Downloads**: 2356 -> 25592 (+986% y/y)
+- **On GitHub**, Docusaurus 1 is used by 6311 projects (+62.9% y/y) while Docusaurus 2 is used by 5039 projects (+1940% y/y)
+
+## Collaboration with Major League Hacking
+
+We have welcomed [Major League Hacking](https://mlh.io/) **open-source interns** for 2 seasons already.
```
Comment (JoelMarcey) [inline website-1.x/blog/2020-01-12-docusaurus-2020-recap.md:80]: Change:

> We have welcomed [Major League Hacking](https://mlh.io/) **open-source interns** for 2 seasons already.

to:

> We have welcomed [Major League Hacking](https://mlh.io/) (MLH) **fellows** for 2 sessions already.

File: website-1.x/blog/2020-01-12-docusaurus-2020-recap.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+- Adding CodeSandbox support
+- Improving the CI pipelines: build size and Lighthouse bots
+
+We look forward to continuing this collaboration in 2021.
+
+## Media
+
+Dmitry Vinnik (dev advocate @ Facebook) explains Docusaurus in [60 seconds](https://www.youtube.com/watch?v=_An9EsKPhp0) or [15min](https://www.youtube.com/watch?v=Yhyx7otSksg) videos.
+
+Rachel Nabors (documentation @ Facebook) talked a bit about Docusaurus in [React Native Radio 178 (podcast)](https://reactnativeradio.com/episodes/178-documenting-react-native-with-rachel-nabors-gWxwySPl), and William Candillon the [React Native website migration (video)](https://www.youtube.com/watch?v=-zhjq2ECKq4).
```
Comment (JoelMarcey) [inline website-1.x/blog/2020-01-12-docusaurus-2020-recap.md:96]: Rachel Nabors (Documentation Engineer @ Facebook)

File: website-1.x/blog/2020-01-12-docusaurus-2020-recap.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+- Help migrate the React Native website from v1 to v2
+- Making the config validation more strict and exhaustive, returning helpful error messages
+- Adding CodeSandbox support
+- Improving the CI pipelines: build size and Lighthouse bots
+
+We look forward to continuing this collaboration in 2021.
+
+## Media
+
+Dmitry Vinnik (dev advocate @ Facebook) explains Docusaurus in [60 seconds](https://www.youtube.com/watch?v=_An9EsKPhp0) or [15min](https://www.youtube.com/watch?v=Yhyx7otSksg) videos.
```
Comment (JoelMarcey) [inline website-1.x/blog/2020-01-12-docusaurus-2020-recap.md:94]: Dmitry Vinnik (Developer Advocate @ Facebook)

**Review by JoelMarcey — APPROVED** [review]
Please make a few minor changes and then this looks great to me!!

Thanks so much for writing this @slorber ☺️

**Comment by HonkingGoose** [comment]
Hi @JoelMarcey 👋 

You are missing out on a great GitHub feature! You can make your improvements easy to implement for the maintainers by using the "suggestions" feature in GitHub.

I suggest you check out the [GitHub docs, adding line comments to a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)
Step number 6 is the good stuff!

By doing it this way, you also get credits as a "co-author" in the Git commit!

I hope you find this useful! 😄 😉 

Greetings,

HonkingGoose

**Comment by lex111** [comment]
I just thought, if someone has a photo of Docusaurus plushie (I don't have one), it would be nice to use it as the main picture of new blog post.

**Comment by slorber** [comment]
I don't think it's easy to get one as people are working remotely.

Apart this one from google that is not very suitable for a banner/social card imho: https://twitter.com/ericnakagawa/status/942819218684182528/photo/1

We are going to publish very very soon so maybe we can find a better one for the next blog post (that is going to be quite soon as i18n is ready :p)

**Comment by JoelMarcey** [comment]
My twitter profile has a Docusaurus plushie picture, if we want to use that? And I think I have another picture lying around too.

**Review by JoelMarcey — APPROVED** [review]
🎉

This is great!

**Pull request merged**